src/Schedule.h

```diff
@@ -441,6 +442,9 @@ struct Dim {
      * loop (see the DimType enum above). */
     DimType dim_type;
 
+    /** The strategy for loop partitioning. */
+    LoopPartitionPolicy loop_partition_policy;
```

Comment:
"loop_" is unnecessary here, because all these properties are loop properties. I suggest:
`Partition partition_policy;`